src/components/HappiGraph/Tex/tex-inheritance.render.ts

```diff
@@ -0,0 +1,565 @@
+import * as d3 from 'd3';
```

Comment:
duplicating the entire code is not ok :)

Reply:
Yes you're right, I've updated this in my recent commit